### PR TITLE
feature/SM-6031-update-docs-after-sandbox-release

### PR DESCRIPTION
**Pull request by Kellym-Kainos**

### CODE REVIEW

File: docs/api-documentation/V2.2/index.md [hunk 1]
```diff
@@ -0,0 +1,3882 @@
...
+  <li>New filter options on comment list pages and exports, including filtering by comment date and topic</li>
+</ol>
+
+## Versions and Changes
+{: .govuk-heading-l #versions}
+
+This section lists any significant changes made to this document (and by extension, the API interfaces themselves) introduced by each recent and upcoming future release.
+{: .govuk-body}
+
+Version 2.2 (06/08/2020):
```
Comment (pwright08) [inline docs/api-documentation/V2.2/index.md:2781]: 2.1 heading has been removed and 2.2 includes 2.1's changes

**Comment by pwright08** [comment]
@Kellym-Kainos Should the main api docs index page (https://departmentfortransport.github.io/street-manager-docs/api-documentation/) be updated too with the new current version on sandbox?

**Comment by Kellym-Kainos** [comment]
@pwright08  yep thanks! just pushed the changes

File: docs/api-documentation/index.html [hunk 2]
```diff
@@ -73,18 +73,18 @@ <h1 class="govuk-heading-m">Releases</h1>
...
 </ul>
 
 <ul class="govuk-list govuk-list--bullet">
   <li>
-    <!-- UPDATE ME to upcoming latest version -->
-    <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V2.1/">
-      Version 2.1 - Due for release to SANDBOX 23/07/2020. Due for release to PRODUCTION 30/07/2020.
+    <!-- UPDATE ME to upcoming stable version if required -->
+    <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V1.25.3/">
+      Version 1.25.3 - Due for release to SANDBOX 06/08/2020. Due for release to PRODUCTION 13/08/2020.
```
Comment (pwright08) [inline docs/api-documentation/index.html:87]: I think we can comment this out for now - as far as I know we'll only release new stable versions as and when we need to

File: docs/api-documentation/index.html [hunk 3]
```diff
@@ -51,17 +51,17 @@ <h1 class="govuk-heading-m">Releases</h1>
...
     </a>
   </li>
 </ul>
 
 <ul class="govuk-list govuk-list--bullet">
   <li>
-    <!-- UPDATE ME to latest/current stable version if required-->
-    <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V1.25.1/">
-      Version 1.25.1 - Released to SANDBOX 09/07/2020. Released to PRODUCTION 16/07/2020.
+    <!-- UPDATE ME to upcoming stable version if required -->
```
Comment (caoimheb) [inline docs/api-documentation/index.html:62]:
```suggestion
    <!-- UPDATE ME to latest stable version if required -->
```

**Pull request merged**